app/controllers/api/v1/web_hooks_controller.rb

```diff
@@ -41,6 +41,12 @@ def fire
 
   private
 
+  def find_rubygem_by_name
+    @rubygem = Rubygem.find_by name: gem_name
+    return if @rubygem || gem_name == WebHook::GLOBAL_PATTERN
+    render plain: "This gem could not be found", status: :not_found
+  end
```

Comment:
We have come full circle :smile: https://github.com/rubygems/rubygems.org/commit/9da4bcf167bd7335751423c024cd11ed08f32349#diff-849974e26c798d3330916f20a296d6d7L50-L62